app/controllers/api/v1/friendly_games_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Api::V1::FriendlyGamesController < ApplicationController
+  def create
+    if User.find_by(api_key: params[:api_key])
+      game = GamesFacade.repurpose(params)
+      render json: FriendlyGameSerializer.new(game), status: :created
+    else
+      render json: {
+        errors: ['please login first']
+        }, status: 500
+      end
+    end
+
+  def update
+    if valid?(params)
+      game = FriendlyGame.find_by(extension: params[:extension])
+      game.current_fen = params[:fen]
+      game.status = params[:status] if params[:status]
+      game.save
+      ActionCable.server.broadcast "friendly_games_channel_#{game.extension}", FriendlyGameSerializer.new(game)
+      render json: { 'get it together chris': 'but actually'}, status: 200
```

Comment:
This needs to be updated for professionalism- as a note, FE does nothing with this response besides verify it's status.